app/src/main/java/com/hover/stax/utils/Constants.java

```diff
@@ -36,4 +40,9 @@
 	final public static String REQUEST_LINK = "request_link";
 	final public static String FRAGMENT_DIRECT = "fragment_direct";
 	final public static String PAYMENT_LINK_SEPERATOR = "-";
+
+	final public static String BOUNTY_EMAIL = "bounty_email";
```

Comment:
Don't use constants for things that are only used in one class. They should be private to the class they are used in. This is only for things that are used in many places, like navigation.